controllers/reviewers.controller.js

```diff
@@ -0,0 +1,31 @@
+class ReviewerController {
```

Comment:
We can park the feature of `reviewer` as a role under the user resource. Consider deleting all related files to `reviewers`.